client/src/modules/inventory/store.ts

```diff
@@ -1,10 +1,44 @@
 import ApiNode from "@/modules/editor/models/ApiNode";
+import { ActionContext } from "vuex";
+import { RootState } from "@/store";
+import { GET } from "@/utility";
+import ApiRelation from "@/modules/editor/models/ApiRelation";
 
 export class InventoryState {
     /**
-     * Replication of the overview item that is dragged into the diagram
+     * Replication of the overview item that is selected or dragged into the diagram
      */
     public selectedNode?: ApiNode;
+
+    /**
+     * First degree neighbors of the currently selected node
+     */
+    public neighbors = [] as Array<ApiNode>;
+
+    /**
+     * Relations for the currently displayed graph
+     */
+    public relations = [] as Array<ApiRelation>;
+
+    /**
+     * Maps the uuid of a node to the id of a diagram shape
+     */
+    public mappedNodes = new Map<string, string | number>();
+
+    /**
+     * Maps the uuid of a relation to the id of a diagram link
+     */
+    public mappedRelations = new Map<string, string | number>();
```

Comment:
Is it really necessary to save these things in the vuex store? It might also be well suited in a separate class :thinking: